scenario/utils/relayPolygonMessage.ts

```diff
@@ -1,53 +1,134 @@
 import { DeploymentManager } from '../../plugins/deployment_manager';
 import { impersonateAddress } from '../../plugins/scenario/utils';
 import { setNextBaseFeeToZero, setNextBlockTimestamp } from './hreUtils';
-import { Event } from 'ethers';
+import { Contract, Event, ethers } from 'ethers';
 
-export default async function relayMumbaiMessage(
+type BridgeERC20Data = {
+  syncData: string;
+  user: string;
+  rootToken: string;
+  amount: bigint;
+};
+
+function tryDecodeStateSyncedData(stateSyncedData: any): BridgeERC20Data | undefined {
+  try {
+    const { syncData } = ethers.utils.defaultAbiCoder.decode(
+      ['bytes32', 'bytes syncData'],
+      stateSyncedData
+    );
+    const { user, rootToken, depositData } = ethers.utils.defaultAbiCoder.decode(
+      ['address user', 'address rootToken', 'bytes depositData'],
+      syncData
+    );
+    const { amount } = ethers.utils.defaultAbiCoder.decode(['uint256 amount'], depositData);
+    return {
+      syncData,
+      user,
+      rootToken,
+      amount
+    };
+  } catch (e) {
+    return undefined;
+  }
+}
+
+export default async function relayPolygonMessage(
   governanceDeploymentManager: DeploymentManager,
   bridgeDeploymentManager: DeploymentManager,
+  startingBlockNumber: number
 ) {
-  const MUMBAI_RECEIVER_ADDRESSS = '0x0000000000000000000000000000000000001001';
-  const EVENT_LISTENER_TIMEOUT = 60000;
+  const POLYGON_RECEIVER_ADDRESSS = '0x0000000000000000000000000000000000001001';
+  const CHILD_CHAIN_MANAGER_PROXY_ADDRESS =
```

Comment:
super nit, but I believe ALL_CAPS is usually reserved for constants (whereas the value here varies depending on the dm's network)

Reply:
Makes sense, done.